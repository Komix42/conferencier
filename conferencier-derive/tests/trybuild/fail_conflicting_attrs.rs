#[derive(conferencier_derive::ConferModule)]
struct ConflictingAttrs {
    #[confer(default = 5, init = "Default::default()")]
    value: i32,
}

fn main() {}
