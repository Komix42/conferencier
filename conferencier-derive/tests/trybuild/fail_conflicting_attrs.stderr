error: #[confer(default = ...)] and #[confer(init = ...)] cannot be combined
 --> tests/trybuild/fail_conflicting_attrs.rs:3:5
  |
3 |     #[confer(default = 5, init = "Default::default()")]
  |     ^
